packages/@react-spectrum/s2/chromatic/Tabs.stories.tsx

```diff
@@ -57,9 +57,9 @@ export const Example = (args: any) => (
 export const Disabled = (args: any) => (
   <Tabs {...args} styles={style({width: 450, height: 144})} disabledKeys={['FoR', 'MaR', 'Emp']}>
     <TabList aria-label="History of Ancient Rome">
-      <Tab id="FoR"><Edit /><Text>Founding of Rome</Text></Tab>
-      <Tab id="MaR">Monarchy and Republic</Tab>
-      <Tab id="Emp">Empire</Tab>
+      <Tab id="FoR" aria-label="Edit"><Edit /><Text>Edit</Text></Tab>
```

Comment:
Should the keys change to match the new values? Why are disabling all the keys?

Reply:
a bit of knowledge about the internal workings that is being tested to make sure that disabling all the keys individually works the same
this is a chromatic story, not fussed about the keys